src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -48,13 +49,12 @@ export const LC = {
     FG_VIEWER_FINDER: 'LC_VIEWER_FINDER',          // mission and general entries group
     FG_PERIOD_FINDER: 'LC_PERIOD_FINDER',          // period finding group for the form
     FG_PERIODOGRAM_FINDER: 'LC_PERIODOGRAM_FINDER',// periodogram finding group
-    FG_TIMESERIES_FINDER: 'LC_TIMESERIES_FINDER',  // result layout panel finding group
     PERIODOGRAM_GROUP: 'LC_PERIODOGRAM_TBL',       // table container, table group
 
-    META_TIME_NAMES: 'timeNames',
-    META_FLUX_NAMES: 'fluxNames',
-    META_ERR_NAMES: 'errorNames',
-    META_MISSION: MetaConst.DATASET_CONVERTER,
+    META_TIME_NAMES: 'ts_timeNames',
+    META_FLUX_NAMES: 'ts_fluxNames',
+    META_ERR_NAMES: 'ts_errorNames',
+    META_MISSION: MetaConst.TS_DATASET,
```

Comment:
Just curious, why is META_MISSION 'TSDatasetId' when the other meta are all 'ts_*'?